src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFOutline.java

```diff
@@ -75,8 +75,14 @@ public PDOutlineItem copyDocumentOutline(int currentPageNumber, PDOutlineItem it
         return item.getNextSibling();
     }
 
-    public int getOutlinePage(PDOutlineItem outlineItem) throws IOException {
-        PDPageDestination dest = (PDPageDestination) outlineItem.getDestination();
-        return dest == null ? -1 : Math.max(dest.retrievePageNumber(), 0);
+    public int getOutlinePage(PDOutlineItem outlineItem) {
+        PDPageDestination dest = null;
+        try {
+            dest = (PDPageDestination) outlineItem.getDestination();
+            return dest == null ? -1 : Math.max(dest.retrievePageNumber(), 0);
+        } catch (IOException e) {
+            System.out.println("Error message: " + e);
```

Comment:
use logger instead of System.out.println

Reply:
included it